src/main/java/core/basesyntax/service/IServiceReport.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service;
+
+public interface IServiceReport {
```

Comment:
```suggestion
public interface ServiceReport {
```
чому ти додаєш І на початок? 
В реальних проектах прийнято до класу імплементації додавати Impl в кінець
Виправ будь ласка інші класи також

Reply:
I - interface